### PR TITLE
Automation tests for Storybook components

### PR DESCRIPTION
**Pull request by DlgSHi**
**Added automation scripts to testing scope:**

- Message,
- Mount in App, 
- Multi Action Button,
- Navigation Bar

- improvements for menu / menuList feature files

### CODE REVIEW

File: cypress/support/step-definitions/message-steps.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Then('Message component is visible', () => {
+  messagePreview().should('be.visible');
+});
+
+Then('Message component is not visible', () => {
+  messagePreview().should('not.exist');
+});
+
+Then('{word} function was called in Actions Tab', () => {
```
Comment (marcinstachowicz) [inline cypress/support/step-definitions/message-steps.js]: click should be parameter

File: cypress/locators/message/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import {
+  MESSAGE_TITLE, MESSAGE_TYPE, MESSAGE_PREVIEW,
+  MESSAGE_CHILDREN,
+  MESSAGE_DISMISS_ICON,
```
Comment (marcinstachowicz) [inline cypress/locators/message/index.js]: is it possible to make import in less lines?

File: cypress/integration/multiActionButton.feature [hunk 3]
```diff
@@ -28,24 +28,23 @@ Feature: Multi Action Button component
...
+    Then Multi Action Button align on preview is "<align>"
+     Examples:
+      | align |
+      | left  |
+      | right |
 
   @positive
   Scenario: Check disabled state of Multi Action Button component
-    When I check disabled checkbox
+    When I disable disabled component
```
Comment (marcinstachowicz) [inline cypress/integration/multiActionButton.feature]: `When I disable MultiActionButton component`

File: cypress/support/step-definitions/multi-action-button-steps.js [hunk 4]
```diff
@@ -28,14 +28,14 @@ Then('Multi Action Button as on preview is {string}', (asProperty) => {
...
-Then('Multi Action Button align on preview is left', () => {
-  multiActionButtonPreview()
-    .should('not.have.class', `${MULTI_ACTION_BUTTON_ALIGN_RIGHT}`);
-});
-
-Then('Multi Action Button align on preview is right', () => {
-  multiActionButtonPreview()
-    .should('have.class', `${MULTI_ACTION_BUTTON_ALIGN_RIGHT}`);
+Then('Multi Action Button align on preview is {string}', (align) => {
+  if (align === 'left') {
```
Comment (marcinstachowicz) [inline cypress/support/step-definitions/multi-action-button-steps.js]: please add comment `// left is default align that's why it's absent inside class`

Comment (DlgSHi) [reply cypress/support/step-definitions/multi-action-button-steps.js]: fixed

File: cypress/integration/multiActionButton.feature (hunk 3 above)
Comment (DlgSHi) [reply cypress/integration/multiActionButton.feature]: fixed

File: cypress/locators/message/index.js (hunk 2 above)
Comment (DlgSHi) [reply cypress/locators/message/index.js]: fixed

File: cypress/integration/multiActionButton.feature [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    Then Multi Action Button align on preview is left
+
+  @positive
+  Scenario: Check right align property of Multi Action Button component
+    When I select align to "right"
+    Then Multi Action Button align on preview is right
+
+  @positive
+  Scenario: Check disabled state of Multi Action Button component
+    When I check disabled checkbox
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/integration/multiActionButton.feature]: fixed

File: cypress/integration/multiActionButton.feature [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+@positive
+  Scenario: Check left align property of Multi Action Button component
+    When I select align to "left"
+    Then Multi Action Button align on preview is left
+
+  @positive
+  Scenario: Check right align property of Multi Action Button component
+    When I select align to "right"
+    Then Multi Action Button align on preview is right
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/integration/multiActionButton.feature]: fixed

File: cypress/support/step-definitions/message-steps.js (hunk 1 above)
Comment (DlgSHi) [reply cypress/support/step-definitions/message-steps.js]: fixed

File: cypress/integration/confirm.feature [hunk 7]
```diff
@@ -4,129 +4,130 @@ Feature: Confirm component
...
-    Then confirm button content on preview is "<confirmLabel>"
-    Examples:
-      | confirmLabel            |
-      | Sample text              |
-      | 1234567890               |
-      | áéíóú¿¡üñ                |
-      | !@#$%^*()_+-=~[];:.,?{}  |
-      | ÄÖÜßäöüß                 |
-      | <>                       |
+  # @positive
```
Comment (marcinstachowicz) [inline cypress/integration/confirm.feature]: forgot to uncomment after testing? ;)

Comment (DlgSHi) [reply cypress/integration/confirm.feature]: yes :), fixed

**Pull request merged**